.github/CODEOWNERS

```diff
@@ -66,6 +66,7 @@
 /src/cdk/a11y/**                                   @jelbourn @devversion
 /src/cdk/accordion/**                              @jelbourn
 /src/cdk/bidi/**                                   @jelbourn
+/src/cdk/clipboard/**                              @jelbourn
```

Comment:
Not sure whether we want any other people to own this.

Reply:
You can add `@xkxx`